### DISCUSSION TITLE
Other page documentation

### DISCUSSION THREAD

**Discussion by soyJDan** [FabricMC/fabric-api#3227] [FAQ] [answered]
Hello!

I have a question, is there any other page that contains the official documentation of FabricMC, as is the official page [FabricMC Wiki](https://fabricmc.net/wiki/start)?

Personally, I find it a bit complex at first glance, and I would like to look for another one that is more interactive with the developer.

**Comment by IMB11** [comment, marked as answer]
https://fabric.moddedmc.wiki/ - non official

**Reply by soyJDan** [reply]
Thanks
